test/random_colorize_pipe_test.dart

```diff
@@ -0,0 +1,42 @@
+@TestOn('browser')
+import 'package:angular2/src/common/pipes/invalid_pipe_argument_exception.dart';
+import 'package:skawa_components/src/pipes/random_colorize_pipe.dart';
+import 'package:test/test.dart';
+
+final Matcher throwsATypeError = throwsA(new isInstanceOf<TypeError>());
+
+final Matcher throwsAnInvalidPipeArgumentException = throwsA(new isInstanceOf<InvalidPipeArgumentException>());
+
+main() {
+  group('RandomColorizePipe | ', () {
+    SkawaRandomColorizePipe pipe;
+    final int intSeed = 1234;
+    final String stringSeed = 'seed';
+    setUp(() => pipe = new SkawaRandomColorizePipe());
+    test('accepts int', () {
+      expect(() {
+        pipe.transform(intSeed);
+      }, returnsNormally);
+    });
+    test('accepts String', () {
+      expect(() => pipe.transform(stringSeed), returnsNormally);
+    });
+    test('throws for types not int or String', () {
+      expect(() => pipe.transform(new Object()), throwsAnInvalidPipeArgumentException);
+    });
+    test('returns same color for the same int seed', () {
+      String a = pipe.transform(intSeed);
+      String b = pipe.transform(intSeed);
+      expect(a, allOf(const isInstanceOf<String>(), b));
+    });
+    test('returns valid css color', () {
+      String a = pipe.transform(intSeed);
+      expect(a, allOf(startsWith('#'), hasLength(lessThanOrEqualTo(7))));
```

Comment:
It has to be equal to either 7 or 4.

Reply:
5 or 9 also valid (and correctod corresponding to this)